book/theme/index.hbs

```diff
@@ -0,0 +1,384 @@
+<!DOCTYPE HTML>
```

Comment:
nit: Is it copied (and modified) from some other repository? If so, it seems good to add a link to the original place to honor the original author and to make future readers clear

Reply:
as mgeisler mentioned, this file comes from the mdbook itself, how about the following:
```html
{{!-- Modified from https://github.com/rust-lang/mdBook/blob/master/src/theme/index.hbs --}}
 ```